Split `BVTest>>testBvViaInt` into signed and unsigned

shingarov opened the pull request.

shingarov committed:
Split BVTest>>testBvViaInt into signed and unsigned

janvrany commented:
LGTM. We also need to increase the limit to make it pass, like I tried in #569. All we need is support for `#timeLimit:` in St/X - the code is there, it just need to go through build pipeline. Once it does, I'll update this PR and merge. I expect this to happen tomorrow morning.

janvrany committed:
[Z3] Increase time limit for `BVTest >> testBvViaSInt` to 15 minutes

Starting with Z3 4.15.1, this test started to take a long time. On (my)
Intel i7-1355U it takes ~5mins. This commit increases time limit to
15 minutes (considering the fact that CI is run in a cloud and not on
dedicated machine so the physical machine may be running other workloads
too).

Hopefully fixes #562.
